MainRepository/Program.cs

```diff
@@ -1,7 +1,68 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find the possible combination of queen in the n*n chess board
+class NQueen {
+   static int Count;
+   //Function to check if two queens attack each other or not
+   // Unicode of queen-\u2655
+   static bool IsSafe (char[,] board, int r, int c) {
+      // Checking the column 
+      for (int i = 0; i < r; i++) {

```

Comment:
```suggestion
      for (int i = 0; i < r; i++) 
           if (board[i, c] == '\u2655') return false;
```
Remove the brackets if the body has single line.